client/components/ArtFilters.js

```diff
@@ -19,8 +19,13 @@ const ArtFilters = ({ artworks, artists, genres, changeFilter }) => {
         <option value="DEFAULT">GENRE</option>
         { genres ?
           genres.map(genre => {
+            const numArts = artworks
+              .map(a => a.genres)
+              .map(a => a.map(g => g.id))
+              .filter(a => a.includes(genre.id))
+              .length;
             return (
-              <option value={ genre.id } key={ genre.id }>{ genre.name } ({ genre.artworks.length })</option>
```

Comment:
I think this does some weird stuff w the filters now if you select both an artist and a genre.